docs/resources/group.md

```diff
@@ -0,0 +1,25 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_group Resource - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Groups are how users are authorized to access Resources. For more information, see Twingate's documentation https://docs.twingate.com/docs/groups.
+---
+
+# twingate_group (Resource)
+
+Groups are how users are authorized to access Resources. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/groups).
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the group
+
+### Read-Only
```

Comment:
Lets add an example of how to use and import groups , like in all other resources 
in order for that to happen you need to add a corresponding file to 
<img width="552" alt="image" src="https://user-images.githubusercontent.com/2689706/163737596-612865c6-58b9-4928-b60c-940307388f63.png">